src/components/dashboard/StakingBanner/index.tsx

```diff
@@ -0,0 +1,128 @@
+import { Typography, Card, SvgIcon, Grid, Button, Box, Stack } from '@mui/material'
+import css from './styles.module.css'
+import Kiln from '@/public/images/common/kiln.svg'
+import StakeIllustrationLight from '@/public/images/common/stake-illustration-light.svg'
+import StakeIllustrationDark from '@/public/images/common/stake-illustration-dark.svg'
+import classNames from 'classnames'
+import { useDarkMode } from '@/hooks/useDarkMode'
+import { useRouter } from 'next/router'
+import NextLink from 'next/link'
+import { OVERVIEW_EVENTS, trackEvent } from '@/services/analytics'
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import ExternalLink from '@/components/common/ExternalLink'
+import { useSanctionedAddress } from '@/hooks/useSanctionedAddress'
+import useIsStakingFeatureEnabled from '@/features/stake/hooks/useIsSwapFeatureEnabled'
+import { AppRoutes } from '@/config/routes'
+import useBalances from '@/hooks/useBalances'
+import { formatUnits } from 'ethers'
+import { TokenType } from '@safe-global/safe-gateway-typescript-sdk'
+
+const LOCAL_STORAGE_KEY_HIDE_WIDGET = 'hideStakingBanner'
+const LEARN_MORE_LINK = ''
+const MIN_NATIVE_TOKEN_BALANCE = 32
+
+const StakingBanner = () => {
+  const isDarkMode = useDarkMode()
+  const router = useRouter()
+  const { balances } = useBalances()
+
+  const nativeTokenBalance = balances.items.find((balance) => balance.tokenInfo.type === TokenType.NATIVE_TOKEN)
+
+  const hasSufficientFunds =
+    nativeTokenBalance != null &&
+    Number(formatUnits(nativeTokenBalance.balance, nativeTokenBalance.tokenInfo.decimals)) >= MIN_NATIVE_TOKEN_BALANCE
+
+  const [widgetHidden = false, setWidgetHidden] = useLocalStorage<boolean>(LOCAL_STORAGE_KEY_HIDE_WIDGET)
+
+  const isStakingFeatureEnabled = useIsStakingFeatureEnabled()
```

Comment:
@schmanu maybe instead of looking whether staking is enabled we could have a separate feature flag for only the banner? This way we can turn it off one day without doing a new release of the app. Or too much? What do you think? We don't do this for the swaps banner now and we'll have to remove it. I think the staking banner is supposed to take the Swap place.

Reply:
I would say its a product question. Not sure if we want to keep the other widgets 🤷

Reply:
Andrei confirmed that the swap banner should be replaced by the new one. I'm on it.